ocis/pkg/runtime/service/service.go

```diff
@@ -96,6 +100,7 @@ func NewService(options ...Option) (*Service, error) {
 	s := &Service{
 		ServicesRegistry: make(serviceFuncMap),
 		Delayed:          make(serviceFuncMap),
+		Optional:         make(serviceFuncMap),
```

Comment:
```suggestion
		Additional:         make(serviceFuncMap),
```
to align with env var and the behavior?

Reply:
done